src/tram-one.js

```diff
@@ -10,7 +10,7 @@ const start = require('./start')
  * @package
  * @public
  * @description
- * Tram-One is a light View Framework that comes with all the dependencies you need to start developing on the web.
+ * Tram-One is a Modern View Framework that has advance features like hooks, observables, and JSX-like template components, all in plain vanilla javascript.
```

Comment:
needed something to test that the badge sizes would be updated, this worked, and this needed to be updated anyways.